### ISSUE TITLE
getProcessOutput doc string should use proper epydoc instead of *this*

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6309]
|[<img alt="hynek's avatar" src="https://avatars.githubusercontent.com/u/41240?s=50" width="50" height="50">](https://github.com/hynek)| @hynek reported|
|-|-|
|Trac ID|trac#6309|
|Type|enhancement|
|Created|2013-02-20 12:14:59Z|
|Branch|https://github.com/twisted/twisted/tree/getProcessOutput-epytext-6309|

This one is a bit OCD, but it caught my eye: twisted.internet.utils.getProcessOutput has the following sentence inside of its doc string:

`The first string should *NOT* be the executable's name.`

I think a `B{NOT}` would be more appropriate.

Attachments:

* [getProcessOutput-use-proper-epydoc.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/6fc/6fc2e14f8fe84696eaffa65fa49563d509edf449/f2d6404bdf4442c386b8003cb8de87c36bef8b30.patch) (564 bytes) - added by hynek on 2013-02-20 12:15:31Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__6309 6309
type__enhancement enhancement
reporter__hynek hynek
priority__lowest lowest
milestone__ 
branch__branches_getProcessOutput_epytext_6309 branches/getProcessOutput-epytext-6309
branch_author__hynek hynek
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1361362499000000 1361362499000000
changetime__1361397093000000 1361397093000000
version__None None
owner__tom_prince tom.prince
cc__hynek
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="tomprince's avatar" src="https://avatars.githubusercontent.com/u/283816?s=50" width="50" height="50">](https://github.com/tomprince)<a name="note_2"></a>|@tomprince commented|
|-|-|

(In [37156]) Branching to getProcessOutput-epytext-6309.

**Comment by twisted-trac** [comment]
|[<img alt="tomprince's avatar" src="https://avatars.githubusercontent.com/u/283816?s=50" width="50" height="50">](https://github.com/tomprince)<a name="note_3"></a>|@tomprince set owner to @tomprince|
|-|-|

This looks good, and the documentation and twistedchecker builder like.

Merging.

**Comment by twisted-trac** [comment]
|[<img alt="tomprince's avatar" src="https://avatars.githubusercontent.com/u/283816?s=50" width="50" height="50">](https://github.com/tomprince)<a name="note_4"></a>|@tomprince set status to `closed`|
|-|-|

(In [37159]) Merge getProcessOutput-epytext-6309: getProcessOutput doc string should use proper epydoc instead of *this*

Author: hynek
Reviewers: tom.prince
Fixes: [#6309](https://github.com/twisted/twisted/issues/6309)

This one is a bit OCD, but it caught my eye: twisted.internet.utils.getProcessOutput has the following sentence inside of its doc string:

The first string should *NOT* be the executable's name.

I think a B{NOT} would be more appropriate.
